src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/advice/ErrorResponse.java

```diff
@@ -3,8 +3,10 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.Setter;
 
 @Getter
+@Setter

```

Comment:
Can we verify the mutation coverage for all the setter methods this will generate